### DISCUSSION TITLE
Filament FileUpload could  not upload image in production

### DISCUSSION THREAD

**Discussion by uniquedigitals** [filamentphp/filament#13541] [Help]
![Capture](https://github.com/filamentphp/filament/assets/85360821/f9d1c9dd-829f-4f20-85b2-a6f77ad521b6)
i was able to upload on local server but liveserver upload doesnt work

**Comment by leandrocfe** [comment]
Did you run `php artisan storage:link` ?
Did you config the APP_URL in .env?
Console errors?

**Reply by uniquedigitals** [reply]
I did

**Reply by fatihtkale** [reply]
did you fix?

**Reply by uniquedigitals** [reply]
No

**Reply by leandrocfe** [reply]
Could you inspect the browser and share the console errors please?

**Reply by fatihtkale** [reply]
i switched from ploi where the error was happening to coolify

**Comment by avinmaster** [comment]
You can look at this one. Change proxy pass settings, especially if you're using Octane.

https://github.com/filamentphp/filament/discussions/9243#discussioncomment-7351604
